command/auth_list.go

```diff
@@ -155,18 +155,40 @@ func (c *AuthListCommand) detailedMounts(auths map[string]*api.AuthMount) []stri
 			replication = "local"
 		}
 
-		out = append(out, fmt.Sprintf("%s | %s | %s | %s | %s | %s | %s | %t | %v | %s",
-			path,
-			mount.Type,
-			mount.Accessor,
-			defaultTTL,
-			maxTTL,
-			mount.Config.TokenType,
-			replication,
-			mount.SealWrap,
-			mount.Options,
-			mount.Description,
-		))
+		pluginName := mount.Type
+		if pluginName == "plugin" {
+			pluginName = mount.Config.PluginName
+		}
+
+		if mount.Type == "plugin" {
```

Comment:
I don't think this branch is needed anymore, we should simply just replace mount.Type with pluginName

Reply:
Also above on line 146 we should s/Type/Plugin/ and remove the existing Plugin column